Heliarch Drills: Add Dialog in case you destroy the Punisher (#9342)
endless-sky/endless-sky: data/coalition/heliarch intro.txt

		government "Heliarch Test Dummy"
		personality disables staying nemesis heroic
		system "Torbab"
		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"

> 		dialog "The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You've failed to simply disable the ship, and doomed the entire crew. It seems you'll not be welcome in Coalition space any longer."

Reviewer: ```suggestion
		dialog "The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You have failed to simply disable the ship, and doomed the entire crew. It seems that you will not be welcome in Coalition space any longer."
```

> 		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"
> 		dialog "The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You have failed to simply disable the ship, and doomed the entire crew. It seems that you will not be welcome in Coalition space any longer."

Reviewer 2: ```suggestion
		ship "Heliarch Punisher (Scrappy)" "Awaiting Adversary"
		on kill
			dialog `The Punisher's captain tries to hail you as the ship collapses around them, but you never get to hear their final cries. You have failed to simply disable the ship, and doomed the entire crew. It seems that you will not be welcome in Coalition space any longer.`
```

Reviewer 2: Here I'm using the `on ---` condition for the `npc` to define a dialogue to appear on the `kill` of the NPC. Otherwise, by default, a dialogue shows up upon completion of the given `npc` condition, in this case being to `disable` it.
